modules/dpu-enable-dpu-hw-offloading.adoc

```diff
@@ -0,0 +1,386 @@
+    - key: machineconfiguration.openshift.io/role
+      operator: In
+      values:
+      - worker
+      - dpu-host
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/dpu-host: ""
+----
+
+.. Run the following command:
++
+[source,terminal]
+----
+$ oc create -f dputenantmachineconfig.yaml
+----
+
+. Label the DPU nodes:
++
+[source,terminal]
+----
+$ oc label node <NODENAME> node-role.kubernetes.io/dpu-host=
+----
+
+. Install and configure the SR-IOV Network Operator:
+
+.. In the {product-title} web console, click *Administration* -> *Namespaces*.
+
+.. Click *Create Namespace*.
+
+.. In the *Name* field, enter `openshift-sriov-network-operator`, and click *Create*.
+
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Select *SR-IOV Network Operator* from the list of available Operators, and click *Install*.
+
+.. On the *Install Operator* page, under *A specific namespace on the cluster*, select *openshift-sriov-network-operator*.
+
+.. Click *Install*.
+
+.. Verify that the SR-IOV Network Operator is installed successfully:
+
+... Navigate to the *Operators* -> *Installed Operators* page.
+
+... Ensure that *SR-IOV Network Operator* is listed in the *openshift-sriov-network-operator* project with a *Status* of *InstallSucceeded*.
++
+[NOTE]
+====
+During installation an Operator might display a *Failed* status.
+If the installation later succeeds with an *InstallSucceeded* message, you can ignore the *Failed* message.
+====
++
+If the operator does not appear as installed, to troubleshoot further:
+
++
+* Inspect the *Operator Subscriptions* and *Install Plans* tabs for any failure or errors under *Status*.
+* Navigate to the *Workloads* -> *Pods* page and check the logs for pods in the
+`openshift-sriov-network-operator` project.
+
+. Add this machine config pool to the SriovNetworkPoolConfig custom resource.
+
+.. Create a file, such as `sriov-pool-config.yaml`, with the following content:
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkPoolConfig
+metadata:
+  name: default
+  namespace: openshift-sriov-network-operator
+spec:
+  ovsHardwareOffloadConfig:
+    name: dpu-host <1>
+----
+<1> The name here is the same as the machine config pool (MCP) name created in step 1.
+.. Apply the configuration:
++
+[source,terminal]
+----
+$ oc create -f sriov-pool-config.yaml
+----
+
+. Create a SriovNetworkNodePolicy to configure the virtual functions (VFs) on the hosts.
+
+.. Save the YAML in the `SriovNetworkNodePolicy.yaml` file:
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policy-mlnx-bf <1>
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: mlnx_bf <2>
+  nodeSelector:
+    node-role.kubernetes.io/dpu-host: "" <3>
+  priority: 99 <4>
+  numVfs: 4 <5>
+  nicSelector: <6>
+    vendor: "15b3" <7>
+    deviceId: "a2d6" <8>
+    pfNames: ['ens1f0'] <9>
```

Comment:
@pliurh  @bn222 
I think this should be it should be "ens1f0#1-3" such that 1 VF is reserved for the management port.

Reply:
yes.